### DISCUSSION TITLE
Descripción proyecto  Heladería

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#58] [General]
Creo esta discusión para hacer la descripción del nuevo proyecto para colocarlo en el pdf que debemos enviar antes del viernes 21/04

**Comment by NataliaAlvarezIspc** [comment]
Venta de productos cerrados de Heladería. El usuario tendrá la posibilidad de elegir helados por pote de cuarto o por medio kilo. Habrá variedad de gustos. Cada opción será de un solo sabor.

**Reply by BraianTroncoso** [reply]
Solamente optaremos por vender Online helados tipo "tentacion" por ende la variedad es limitada.

**Comment by rpgrca** [comment]
En verdad como son artículos cerrados podemos vender potes de 1 litro con tres sabores (frutilla, vainlla, dulce de leche por ejemplo), eso no aumenta la complejidad. No tenemos una búsqueda avanzada así que solamente buscará por palabras clave y no necesariamente filtrar por sabores.

**Reply by Brian-Almada** [reply]
Claro, es como que no elige el cliente los sabores que tiene el pote, ya vienen así por default. No?

**Reply by rpgrca** [reply]
Sí, el pote ya tiene los sabores y no se puede cambiar, puede ser un sabor, dos sabores o tres sabores.
